meson.build

```diff
@@ -1119,32 +1119,17 @@ libfdisk = dependency('fdisk',
                       required : get_option('fdisk'))
 conf.set10('HAVE_LIBFDISK', libfdisk.found())
 
-want_passwdqc = get_option('passwdqc')
-want_pwquality = get_option('pwquality')
-if want_passwdqc == 'true' and want_pwquality == 'true'
-        error('passwdqc and pwquality cannot be requested simultaneously')
-endif
-
-if want_pwquality != 'false' and want_passwdqc != 'true' and not skip_deps
-        libpwquality = dependency('pwquality',
-                                  version : '>= 1.4.1',
-                                  required : want_pwquality == 'true')
-        have = libpwquality.found()
-else
-        have = false
-        libpwquality = []
+feature = get_option('pwquality').disable_auto_if(get_option('passwdqc').enabled())
+libpwquality = dependency('pwquality',
+                          version : '>= 1.4.1',
+                          required : feature)
+have = libpwquality.found()
+if not have
+        libpwquality = dependency('passwdqc',
```

Comment:
Please add a short comment that `libpwquality` is also used for `passwdqc`.
Though, we never pass it to any executables or libraries.